wfe/wfe.go

```diff
@@ -344,6 +352,31 @@ const (
 	unknownKey = "No registration exists matching provided key"
 )
 
+func extractJWSKey(body string) (*jose.JsonWebKey, *jose.JsonWebSignature, error) {
```

Comment:
It looks like you removed the `wfe.stats.Inc("Errors.xxx")` calls for `UnableToParseJWS`, `TooManyJWSSignaturesInPOST`, etc etc. Was that on purpose? Do we track these anywhere?